code/src/App.js

```diff
@@ -1,9 +1,88 @@
-import React from 'react'
+import React, { useState } from 'react';
+
+// All components for questions 
+import Question1 from './components/Question1';
+import Question2 from './components/Question2';
+import Question3 from './components/Question3';
+import Summary from './components/Summary';
+
+
 
 export const App = () => {
+// All states for each component
+const [name, setName] = useState('');
+const [investmentTerm, setInvestmentTerm] = useState('');
+const [investmentType, setInvestmentType] = useState();
+const [showSummary, setShowSummary] = useState();
+
+
+
+// Event Handlers
+const handleNameChange = (e) => {
+  console.log(e.target.value);
+  setName(e.target.value);
+}
+
+const handleInvestmentTermChange = (e) => {
+  console.log(e.target.value);
+  setInvestmentTerm(e.target.value);
+}
+
+const handleInvestmentTypeChange = (e) => {
+  console.log(e.target.value);
+  setInvestmentType(e.target.value);
+}
+
+const handleSubmit = event => {
+  event.preventDefault();
+  setShowSummary(true);
+};
+
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+        <div className="form-container">
+        <span className="money-emoji" role="img" aria-label="Money Bag Emoji">💰</span>
+        <h1>Welcome to Investor Check  
+            <span role="img" aria-label="Checkbox Emoji">✅ </span> 
+        </h1>
+        <p>Double check your investment goals   
+            <span role="img" aria-label="Smile Face Emoji">🤓 </span> 
+        </p>
+        </div>
+
+
+        {!showSummary && (
+        <form onSubmit={handleSubmit}>
+
+          <Question1 
+          username={name} 
+          onNameChange={handleNameChange} 
+          />
+
+
+          <Question2
+           value={investmentTerm}
+           onInvestmentTermChange={handleInvestmentTermChange}
+          />
+
+          <Question3
+          checked={Question3 === investmentType}
+          onInvestmentTypeChange={handleInvestmentTypeChange}
+         
+          
+          />
+       
+        <button className="submit" type="Submit">Submit</button>
+
+        </form>
+    )}
+
+      {showSummary && (
+        <Summary name={name} checked={investmentType} value={investmentTerm} />
+      )}
```

Comment:
One solution to render one question at a time could be this: 

First, you'll need to add a hook for the section, starting at 1:
`const [section, setSection] = useState(1);`

This state will be changed by clicking on a button:
`const onButtonClick = () => {
    setSection(section + 1);
  };`

And this button will be rendered with each question, except for the last one, where your submit button will be rendered.

This is what will be rendered (minus the header). I've excluded the props for the questions and the summary components, don't forget them ;)

```
if (!showSummary) {
    return (
      <div>
        {section === 1 && (
          <div>
            <Question1 />
            <button onClick={onButtonClick}>Next Question</button>
          </div>
        )}

        {section === 2 && (
          <div>
            <Question2 />
            <button onClick={onButtonClick}>Next Question</button>
          </div>
        )}

        {section === 3 && (
          <div>
            <Question3 />
            <button onClick={handleSubmit}>Submit</button>
          </div>
        )}
      </div>
    );
  } else if (showSummary) {
    return <Summary />;
  }
```

This solution builds on what you've already used with the showSummary hook. But there is one downside to it, your validation won't work anymore - it's working right now because it's basic HTML validation within a form, but now your question won't be in a form anymore. You could solve that by adding custom validation (have a look at Van's lecture again), by writing some extra logic that's executed when the next question button is clicked - basically checking if the value of the current input is empty or not.